<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormChat</class>
 <widget class="QWidget" name="FormChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QTextEdit" name="textEditChat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditHistory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChatSent">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send(&amp;S)</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetFile" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="tableWidgetRecvFileList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetSendFileList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>171</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelChatClientValue">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelChatNameValue">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelChatHostValue">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelChatGroupValue">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>37</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelChatClient">
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelChatName">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelChatHost">
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelChatGroup">
      <property name="text">
       <string>Group</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
